### ISSUE TITLE
Create rule S6855 (`jsx-a11y/media-has-caption`): Media elements should have captions

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#4381]
[plugin rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/media-has-caption.md#jsx-a11ymedia-has-caption)
